pkg/parser/extended_syntax_parser.y

```diff
@@ -110,6 +112,16 @@ type EvaluateClause struct {
 	EvaluateInto  string
 }
 
+type MathProgClause struct {
+	// Sense can be MAXIMIZE or MINIMIZE
+	Sense string
+	Objective *Expr
+	Constrants ExprList
+	SolveAttrs Attributes
+	Solver string
+	SolveResult string
```

Comment:
SolveResult => Result

Reply:
Changed to `OptimizeInto` to be the same format of other clause structs like `EvaluateClause`.